Gemfile.lock

```diff
@@ -388,6 +388,8 @@ GEM
     kitchen-salt (0.7.2)
       hashie (>= 3.5)
       test-kitchen (>= 1.4)
+    kitchen-vagrant (1.11.0)
+       test-kitchen (>= 1.4, < 4)
```

Comment:
There's actually one space too many here:

```suggestion
    kitchen-vagrant (1.11.0)
      test-kitchen (>= 1.4, < 4)
```

But not to worry, `Gemfile.lock` is updated every week, so this will be fixed soon enough.

Reply:
OK, it did matter:

* https://gitlab.com/saltstack-formulas/consul-formula/-/pipelines/533392466

Fixed it in 84e5f1416bf8b2861824f8cd3bb1b49b2cd228dc and the CI is passing again:

* https://gitlab.com/saltstack-formulas/consul-formula/-/pipelines/533394424